src/Auth0/Login/Repository/Auth0UserRepository.php

```diff
@@ -37,7 +37,7 @@ public function getUserByUserInfo(array $userInfo) : Authenticatable
     public function getUserByIdentifier($identifier) : ?Authenticatable
     {
         // Get the user info of the user logged in (probably in session)
-        $user = \App::make('auth0')->getUser();
+        $user = \Illuminate\Support\Facades\App::make('auth0')->getUser();
```

Comment:
This appears to be throwing an error with our tests. Any ideas on how to work around this?